inhus/Goal goal
---
inhus/Action[] actions
